scripts/load_data_to_sqlite.py

```
@@ -0,0 +1,100 @@
+def main():
+    # Define paths
+    json_path = Path("assets/processed_rulings_v3_ai_enriched.json")
+    db_path = Path("abyssal_tome.db")
+
+    # Check if JSON file exists
+    if not json_path.exists():
+        logging.error(f"JSON file not found at {json_path}")
+        return
+
+    # Create engine and session
+    engine = create_engine(f"sqlite:///{db_path}")
```

Ruff (0.11.9)

`error()` call on root logger

(LOG015)